roles/common/defaults/main.yml

```diff
@@ -257,7 +262,11 @@ gi_patches:
   - { category: "RU", base: "19.3.0.0.0", release: "19.7.0.0.200414", patchnum: "30783556", patchfile: "p30783556_190000_Linux-x86-64.zip", patch_subdir: "/30899722", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
   - { category: "RU", base: "19.3.0.0.0", release: "19.8.0.0.200714", patchnum: "31326369", patchfile: "p31326369_190000_Linux-x86-64.zip", patch_subdir: "/31305339", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
   - { category: "RU", base: "19.3.0.0.0", release: "19.9.0.0.201020", patchnum: "31720429", patchfile: "p31720429_190000_Linux-x86-64.zip", patch_subdir: "/31750108", prereq_check: FALSE, method: "opatchauto apply", ocm: FALSE, upgrade: FALSE }
+# Warning: GIRU 32126842 for 19.10.0.0.210119 has critical regressions and password protected. Development recommends 19.11.0.0.210420 Patch 32545008 instead of the JAN 2021 quarter RU
```

Comment:
If .210119 isn't recommended to install, shouldn't we comment out that one rather than .210420?

Reply:
I can comment out the line for `19.10.0.0.210119`.
And also, will let the `19.11.0.0.210420` RU stay commented out, as the install-oracle.sh` breaks with 19.11's issue applying OJVM along with RU.
https://mikedietrichde.com/2021/04/22/oracle-19c-installation-with-19-11-0-ru-ojvm-and-some-other-fixes/

Reply:
Done
(will be pushed in the next branch refresh)